adaptivefiltering/dataset.py

```diff
@@ -167,8 +174,24 @@ def restrict(self, segmentation):
         from adaptivefiltering.pdal import PDALInMemoryDataSet
 
         dataset = PDALInMemoryDataSet.convert(self)
+
         return dataset.restrict(segmentation)
 
+    def convert_georef(self, spatial_ref_out="EPSG:4326", spatial_ref_in=None):
```

Comment:
Is this really a user-facing API? When would the user explicitly call it? And if so, when is specifying `spatial_ref_in` an option?

Reply:
I thought it could be helpful, if one wants to export the data to another program or something like that. 
The input probably never going to be used, bu tin cases like our current minimalist dataset it would could be handy to manually override missing or faulty metadata.
I don't expect it to be used often, but maybe not having it will be a problem for someone at some point.